modules/percolator/src/main/java/org/elasticsearch/percolator/PercolatorHighlightSubFetchPhase.java

```diff
@@ -87,7 +87,7 @@ public void process(HitContext hit) throws IOException {
                             continue;
                         }
 
-                        SearchHighlightContext highlight = new SearchHighlightContext(fetchContext.highlight().fields());
+                        SearchHighlightContext highlight = new SearchHighlightContext(fetchContext.highlight().fields(), null);
```

Comment:
@romseygeek Have a look at this change, do you think is correct to set original query to null for percolate queries? From what I read online for percolate query it seems to be the case, yet I am not 100% sure.